src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/IntegrationTestBase.kt

```diff
@@ -67,6 +68,11 @@ class IntegrationTestBase {
 
   fun probationUrl(crn: String) = "/probation-cases/$crn"
 
+  @BeforeEach
+  fun beforeEachIntTest() {
```

Comment:
Moved to integration test base as searchIntTest require this to succeed locally on multiple runs